cmd/krew/cmd/internal/fetch_tag.go

```diff
@@ -36,6 +36,9 @@ func FetchLatestTag() (string, error) {
 	if err != nil {
 		return "", errors.Wrapf(err, "could not GET the latest release")
 	}
+	if response.StatusCode != http.StatusOK {
+		return "", errors.Errorf("expected HTTP status 200 OK, got %s", response.Status)
```

Comment:
can we do %d as well (and use response.StatusCode)?

Reply:
I think the status code is part of the string. For example for status 200, its `"200 OK"`.